### ISSUE TITLE
Need extra use case of "our choice" for database

### ISSUE THREAD

**Issue by abg28** [mdholbrook/bme590final#36]
<img width="870" alt="screen shot 2018-12-06 at 9 37 03 pm" src="https://user-images.githubusercontent.com/40406627/49624426-5a9aa280-f99f-11e8-8663-e6900bcaba51.png">

Any ideas?

**Comment by mdholbrook** [comment]
Maybe we can store total compute time for each user?

**Comment by abg28** [comment]
As in the sum of the latencies for all previous user actions?

**Comment by mdholbrook** [comment]
Yeah maybe. I was brainstorming ideas. Total computing time may be something a server would want to track for each user.
